src/components/Header/Header.js

```diff
@@ -0,0 +1,23 @@
+import { StyleSheet, View } from "react-native";
+import HeaderTitle from "./HeaderTitle";
+import { ProfileImage } from "../ProfileImage/ProfileImage";
+
+const Header = () => {
+  return (
+    <View style={styles.header}>
+      <ProfileImage />
```

Comment:
can we pass the image in props in the "ProfileImage" component? Because we will be getting the image here from the database.

Reply:
Sure. Thanks for pointing this out.